lib/services/ios-project-service.ts

```diff
@@ -844,6 +848,41 @@ We will now place an empty obsolete compatability white screen LauncScreen.xib f
 		return Promise.resolve();
 	}
 
+	public checkForChanges(changesInfo: IProjectChangesInfo, options: IProjectChangesOptions, projectData: IProjectData): void {
+		const nextCommandProvisionUUID = options.provision;
+
+		if (typeof nextCommandProvisionUUID === "string") {
+			const pbxprojPath = this.getPbxProjPath(projectData);
+
+			// Check if the native project's signing is set to the provided provision...
+			let signingChanged = false;
+			if (this.$fs.exists(pbxprojPath)) {
+				const xcode = Xcode.open(pbxprojPath);
+				const signing = xcode.getSigning(projectData.projectName);
+				if (signing && signing.style === "Manual") {
+					for (let name in signing.configurations) {
+						let config = signing.configurations[name];
+						if (config.uuid !== nextCommandProvisionUUID && config.name !== nextCommandProvisionUUID) {
+							signingChanged = true;
+							break;
+						}
+					}
+				} else {
+					signingChanged = true;
```

Comment:
Can we have a comment or something here? It's not clear to me, why if the signing is not Manual it means it's changed?